README.md

```diff
@@ -1,38 +1,47 @@
-
-* Doesn't require nightly build (works on NeoVim 0.4.x)
-* Comments each line, rather than adds block comments; making it easier to toggle code when debugging
-* Uses the built in **commentstring** buffer option to define comment markers
-* Where a marker doesn't have a **space** character as padding this is added, configurable
-** This can be disabled in the options, see below but if useful when workig with numerous linters
-* Supports motions
-
-When the plugin is called it works out the range to comment/uncomment; if all lines in the given range are commented then it uncomments, otherwise it comments the range. This is useful when commenting a block out for testing with a real like comment in it; as for the plugin a comment is a comment.
-
-# Usage
+Toggle comments in Neovim, using built in **commentstring** filetype option;
+written in Lua. Without a doubt this plugin **is not required** and is a rip off
+of [TPope's Commentary](https://github.com/tpope/vim-commentary) with less
+features! What makes this plugin stand out over the numerous other comment
+plugins written in Lua are:
+
+- Comments each line, rather than adds block comments; making it easier to
+  toggle code when debugging
+- Uses the built in **commentstring** buffer option to define comment markers
+- Where a marker doesn't have a **space** character as padding this is added,
+  configurable \*\* This can be disabled in the options, see below but if useful
+  when working with numerous linters
+- Supports motions
+- Dot `.` repeatable
+
+When the plugin is called it works out the range to comment/uncomment; if all
+lines in the given range are commented then it uncomments, otherwise it comments
+the range. This is useful when commenting a block out for testing with a real
+like comment in it; as for the plugin a comment is a comment.
+
+## Usage
 
 Either use the command `CommentToggle`, e.g.:
 
-* `CommentToggle` comment/uncomment current line
-* `67,69CommentToggle` comment/uncomment a range
-* `'<,'>CommentToggle` comment/uncomment a visual selection
+- `CommentToggle` comment/uncomment current line
+- `67,69CommentToggle` comment/uncomment a range
+- `'<,'>CommentToggle` comment/uncomment a visual selection
 
 Or use the default mappings:
 
-* `gcc` comment/uncomment current line, this does not take a count, if you want a count use the `gc{count}{motion}`
-* `gc{motion}` comment/uncomment selection defined by a motion:
-** As lines are commented, any comment toggling actions will default to a linewise.
-** `gcc` comment/uncomment current line
-** `gcip` comment/uncomment a paragraph
-** `gc4w` comment/uncomment current line
-** `gc4j` comment/uncomment 4 lines below the current line
+- `gcc` comment/uncomment current line, this does not take a count, if you want
+  a count use the `gc{count}{motion}`
+- `gc{motion}` comment/uncomment selection defined by a motion: \*\*As lines are
```

Comment:
again why change these list types/

Reply:
I am not sure if your intention here was to have indented list, `**` does not do that though.
Without my changes this block looks like:
<img width="847" alt="Screenshot 2021-07-24 at 15 57 57" src="https://user-images.githubusercontent.com/69750637/126870711-b1ad3ae3-3770-4202-9d10-ecb1fbdb2b38.png">
with them it changes to
<img width="978" alt="Screenshot 2021-07-24 at 15 58 33" src="https://user-images.githubusercontent.com/69750637/126870732-b71f68f1-3da3-488f-aab4-f0adde108f01.png">

Reply:
Ah, just noticed that I have omitted one `**` there. Will hold off fixing it for now until you give me green light. :)

Reply:
top banana